osu.Game/Overlays/Profile/UserGraph.cs

```diff
@@ -118,23 +118,21 @@ private void redrawGraph()
         protected virtual void ShowGraph() => graph.FadeIn(FADE_DURATION, Easing.Out);
         protected virtual void HideGraph() => graph.FadeOut(FADE_DURATION, Easing.Out);
 
-        public ITooltip GetCustomTooltip() => GetTooltip();
+        public ITooltip<UserGraphTooltipContent> GetCustomTooltip() => new UserGraphTooltip();
 
-        protected abstract UserGraphTooltip GetTooltip();
-
-        public object TooltipContent
+        public UserGraphTooltipContent TooltipContent
         {
             get
             {
                 if (data == null || hoveredIndex == -1)
-                    return null;
+                    return default;
```

Comment:
`UserGraphTooltipContent` is a class, so `default == null`, no? Unsure what this change does other than add confusion.

Reply:
Oh, I intended that to be a `struct`.